nemo/utils/exp_manager.py

```diff
@@ -96,7 +96,10 @@ class CallbackParams:
     save_top_k: Optional[int] = 3
     save_weights_only: Optional[bool] = False
     mode: Optional[str] = "min"
+    auto_insert_metric_name: bool = True
     every_n_epochs: Optional[int] = 1
+    every_n_train_steps: Optional[int] = None
+    train_time_interval: Optional[timedelta] = None
```

Comment:
Looks good. Maybe we should look into some way of testing the callbacks data class (if it exists in PTL) against exp managers checkpoint subconfig similar to how we check PTL trainer args against our trainer config.

Reply:
It doesn't exist as far as I know, I think it's just the parameters of the ModelCheckpoint init function. But I'm not 100% sure, I haven't looked at it in depth.